app/parse.py

```diff
@@ -8,8 +15,66 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def write_quotes_to_csv(csv_path: str, quotes: [Quote]) -> None:
+    with open(csv_path, "w", encoding="utf-8") as file:
+        writer = csv.writer(file)
+        writer.writerow(QUOTE_FIELDS)
+        writer.writerows([astuple(quote) for quote in quotes])
+
+
+def parse_single_quote(quote_soup: Tag) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[tag.text for tag in quote_soup.select(".tags > .tag")]
+    )
+
+
+def is_there_next_page(soup: BeautifulSoup) -> bool:
+    pagination = soup.select_one(".pager > .next > a")
+
+    if pagination is None:
+        return False
+
+    return True
```

Comment:
You could simplify this check
```suggestion
    return bool(pagination)
```